Xamarin.Forms.sln

```diff
@@ -221,7 +221,6 @@ Global
 		{271193C1-6E7C-429C-A36D-3F1BE5267231}.Debug|ARM.ActiveCfg = Debug|Any CPU
 		{271193C1-6E7C-429C-A36D-3F1BE5267231}.Debug|ARM.Build.0 = Debug|Any CPU
 		{271193C1-6E7C-429C-A36D-3F1BE5267231}.Debug|iPhone.ActiveCfg = Debug|Any CPU
```

Comment:
Could you undo this changes in the solution?

Reply:
I will @jsuarezruiz , thank you very much for the comments.